apycula/tiled_fuzzer.py

```diff
@@ -253,16 +293,19 @@ def run_pnr(mod, constr, config):
         pnr.opt = tmpdir+"/pnr.cfg"
         with open(tmpdir+"/run.tcl", "w") as f:
             pnr.write(f)
+
         subprocess.run([gowinhome + "/IDE/bin/gw_sh", tmpdir+"/run.tcl"])
-        # print(tmpdir); input()
+        #print(tmpdir); input()
         try:
+            attrs = constr.attrs;
             return (*bslib.read_bitstream(tmpdir+"/impl/pnr/top.fs"), \
                    list(read_posp(tmpdir+"/impl/pnr/top.posp")), \
-                   config)
+                   config, attrs, \
+                   read_err_log(tmpdir+"/impl/pnr/top.log"))
```

Comment:
Actually I was thinking of this one. Or maybe both could be named tuples. Or not, it's not super important.

Reply:
done